src/bundle/Resources/public/scss/ui/modules/universal-discovery/mixins/_filters-panel.scss

```diff
@@ -0,0 +1,104 @@
+@mixin c-filters-panel($filterRowCssClass) {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    padding-bottom: calculateRem(24px);
+
+    &__header {
+        display: flex;
+        position: sticky;
+        top: 0;
+        background-color: $ibexa-color-white;
+        z-index: 2;
+        align-items: center;
+        justify-content: space-between;
+        padding: calculateRem(16px) calculateRem(12px);
+        border-bottom: calculateRem(1px) solid $ibexa-color-light;
+    }
+
+    &__header-content {
+        font-size: $ibexa-text-font-size-large;
+        font-weight: 600;
+        margin-right: calculateRem(8px);
+    }
+
+    &__header-actions {
+        display: flex;
+        flex-wrap: nowrap;
+    }
+
+    &__collapsible {
+        & + #{$filterRowCssClass} {
```

Comment:
couldn't we achieve it by setting some constant classname that would just need to be used in html?

Just looking for other solution, current one is probably ok, just looks strange for me (probably needs to get used to passing classname as variable)

Reply:
I guess you cannot generate a CSS selector based on the CSS variable set on the DOM element?